styles/style.css

```diff
@@ -530,11 +531,19 @@ h2 p {
     justify-content: center;
 }
 
-.answer:hover{
+.answer:not(:focus):hover {     /* only apply hover styling when the div is not focused */
     background-color: white;
     box-shadow: 0px 0px 10px 3px rgba(255, 255, 255, 0.5);
 }
 
+#quizAnswers-wrapper .correct {
+    background-color: var(--correctAnswer);
+}
+
```

Comment:
Stylowanie po ID,  Czy jest możliwość zmiany ?

Reply:
Pracowałam już na kodzie @Yimmir i kontenery na quiz mają tylko ID, bez klas. Nie mogłam tego ostylować  tylko za pomocą selektorów '.correct' i '.incorrect', bo potrzebna była większa specyficzność niż .answer:hover. Dopisałam tam jeszcze .answer:not(focus):hover, co jednak jest bez sensu, bo stylowanie na klasach correct i incorrect trwa dłużej niż na sam klik. Ale zrobię to za pomocą JS i dodam klasę 'no-hover' przy zaznaczeniu odpowiedzi i wstawie ją zamiast focus. Powinno działać już bez ID. Dzięki!